### ISSUE TITLE
✨ Feature Suggestion | 

### ISSUE THREAD

**Issue by m0rcq** [privacytoolsIO/privacytools.io#1790]
## Description

Encrypted Domain Name System (DNS) Resolvers addition.

Please consider adding DNSlify's public resolvers to diversify the privacy options available to Internet users.

DNS Provider: DNSlify
Server Locations: Anycast (based in UK)
Privacy Policy: https://www.dnslify.com/privacy/
Type: Commercial
Logging: No
Protocols: DoH, DoT
DNSSEC: Yes
QNAME Minimization: Yes
Filtering: Based on server choice
Hosting Provider: Self

Regards,

**Comment by blacklight447-ptio** [comment]
are you affiliated with dnslify?

**Comment by m0rcq** [comment]
Yes, please check my bio and www.dnslify.com web page footer - we (peeriX) operate DNSlify.

**Comment by danarel** [comment]
> Yes, please check my bio and www.dnslify.com web page footer - we (peeriX) operate DNSlify.

We shouldn’t have to check your bio. We have a template created for recommendations that you didn’t fill out. That template lets us know if you have an affiliation with the product.
